build/ci/Makefile

```diff
@@ -36,6 +36,7 @@ ci-internal: ci-build-image
 		-v /var/run/docker.sock:/var/run/docker.sock \
 		-v $(ROOT_DIR):$(GO_MOUNT_PATH) \
 		-w $(GO_MOUNT_PATH) \
+		--network="host" \
```

Comment:
Why do we need this?

Reply:
Otherwise we can't access `kind` cluster within container